### PR TITLE
fix(test): fix travis and tests

### PR DESCRIPTION
**Pull request by ddysher**
**What this PR does / why we need it**:

Fix travis and tests.

**Which issue(s) this PR fixes** *(optional, close the issue(s) when PR gets merged)*:

**Special notes for your reviewer**:

/cc @kdada @Jimexist 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: hack/verify_boilerplate.py [hunk 1]
```diff
@@ -91,7 +91,7 @@ def file_passes(filename, refs, regexs):  # pylint: disable=too-many-locals
         if year.search(datum):
             return False
 
-    # Replace all occurrences of the regex "2017|2016|2015|2014" with "YEAR"
```
Comment (kdada) [inline hack/verify_boilerplate.py]: 2014 - 2016 could be removed.

File: hack/verify_boilerplate.py [hunk 2]
```diff
@@ -153,7 +153,7 @@ def get_regexs():
     regexs = {}
     # Search for "YEAR" which exists in the boilerplate, but shouldn't in the real thing
     regexs["year"] = re.compile('YEAR')
-    # dates can be 2014, 2015, 2016 or 2017, company holder names can be anything
```
Comment (kdada) [inline hack/verify_boilerplate.py]: ditto

**Comment by ddysher** [comment]
@kdada Done

**Pull request merged**